conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -6,6 +6,10 @@ jobs:
 - job: linux
   {{ azure_yaml|indent(2) }}
   steps:
+{%- if clone_depth is not none %}
+  - checkout: self
+    fetchDepth: {{ clone_depth }}
```

Comment:
What does a fetch depth of zero mean for azure?

Reply:
I thought I saw some documentation of this somewhere, but now I can't find it. I'll just test a value of zero and report back.

Reply:
I can confirm that `fetchDepth: 0` works on Azure to set no limit. The resulting git fetch command omits the --depth argument.